lib/srp.js

```diff
@@ -0,0 +1,273 @@
+import { BigNumber, Modulus } from 'asmcrypto.js/dist_e8/bignum/bignum';
```

Comment:
```suggestion
import { BigNumber, Modulus } from 'asmcrypto.js/dist_es8/bignum/bignum';
```
Sorry, I made a typo here :S Don't forget to test my suggestions before merging this PR because I clearly didn't  😅